policies/default_user_policy.json

```diff
@@ -2,12 +2,12 @@
   "Version": "2012-10-17",
   "Statement": [
     {
-      "Sid": "user_example",
-      "Effect": "Allow",
+      "Sid": "DefaultRole",
```

Comment:
Nit: I think this is a copy-paste leftover from `policies/default_role.json` and should be `DefaultPolicy`.

Reply:
fixed